flink-ml-parent/flink-ml-lib/src/main/java/org/apache/flink/ml/operator/common/dataproc/vector/VectorToColumnsMapper.java

```diff
@@ -0,0 +1,112 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.apache.flink.ml.operator.common.dataproc.vector;
+
+import org.apache.flink.api.common.typeinfo.TypeInformation;
+import org.apache.flink.api.common.typeinfo.Types;
+import org.apache.flink.ml.api.misc.param.Params;
+import org.apache.flink.ml.common.linalg.DenseVector;
+import org.apache.flink.ml.common.linalg.SparseVector;
+import org.apache.flink.ml.common.linalg.Vector;
+import org.apache.flink.ml.common.mapper.Mapper;
+import org.apache.flink.ml.common.utils.OutputColsHelper;
+import org.apache.flink.ml.common.utils.TableUtil;
+import org.apache.flink.ml.params.dataproc.vector.VectorToColumnsParams;
+import org.apache.flink.table.api.TableSchema;
+import org.apache.flink.types.Row;
+import org.apache.flink.util.Preconditions;
+
+import java.util.Arrays;
+
+/**
+ * This mapper maps vector to table columns, and the table is created with the first
```

Comment:
```suggestion
 * This is a data preprocessing function that transforms {@link Vector}s into {@link Table} columns.
 *
 * <p>Table is created with the first colSize value of the vector.
 * 
 * <p>For sparse vector without given size, it will be treated as vector with infinite size.
 * ...
```